### PR TITLE
build kanto suite-connector and service configuration, add README

### PR DESCRIPTION
**Pull request by frankplus**
- add Kanto suite-connector as git submodule
- add build scripts and build metadata to build Kanto suite-connector in the OpenHarmony build system
- add README
- add service to start kanto at system startup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+This application aims to demonstrate how kanto components can be built using gn. 
+Of course building go applications using this tooling does not bring any benefits 
+on its own. This approach can be used to integrate these components in existing builds.
+
+## Integrate into Oniro-OpenHarmony build system
+- First of all, obtain the source code following [Oniro docs](https://docs.oniroproject.org/quick-build.html#obtaining-the-source-code).
+- Secondly, clone this repository into `//third_party/kanto`
+
+```console
+$ git clone --recurse-submodules  https://github.com/frankplus/third_party_kanto.git ./third_party/kanto
```
Comment (Stefan-Schmidt) [inline README.md]: This should change to the new URL

File: README.md [hunk 2]
```diff
@@ -1,59 +1,106 @@
...
+## Prerequisites
+Before proceeding, ensure you have obtained the Oniro-OpenHarmony source code as per the [Oniro documentation](https://docs.oniroproject.org/quick-build.html#obtaining-the-source-code).
+
+## Integration Steps
+### Cloning Repositories
+Clone the Kanto and Mosquitto repositories into the `//third_party` directory to incorporate them into the Oniro-OpenHarmony system:
 
 ```console
 $ git clone --recurse-submodules  https://github.com/frankplus/third_party_kanto.git ./third_party/kanto
+$ git clone --recurse-submodules  https://github.com/frankplus/third_party_mosquitto.git ./third_party/mosquitto
```
Comment (Stefan-Schmidt) [inline README.md]: Same for this URL

File: README.md [hunk 3]
```diff
@@ -1,59 +1,106 @@
...
+diff --git a/rk3568/config.json b/rk3568/config.json
+index ea2d588..b6a8dc0 100755
+--- a/rk3568/config.json
++++ b/rk3568/config.json
+@@ -9,7 +9,7 @@
+   "api_version": 8,
+   "enable_ramdisk": true,
+   "enable_absystem": false,
+-  "build_selinux": true,
++  "build_selinux": false,
```
Comment (Stefan-Schmidt) [inline README.md]: Do we still need to disable this with the  "secon" : "u:r:kanto:s0" change in the next commit?

File: README.md (hunk 2 above)
Comment (frankplus) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (frankplus) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (frankplus) [reply README.md]: updated instructions

**Review by Stefan-Schmidt — APPROVED** [review]

**Pull request merged**